integration/monai/examples/spleen_ct_segmentation/README.md

```diff
@@ -53,6 +53,15 @@ and assign them to the appropriate GPU device using the `--gpu` argument.
 python3 -u -m nvflare.private.fed.app.simulator.simulator job --workspace /tmp/nvflare/sim_spleen_ct_seg --threads 2 --clients site-1,site-2 --gpu 0,1
```

Comment:
there could also be cases where people want multiple threads and single GPU (if they have just one GPU but CPU is powerful).

then they need to do something like: -t 2 -n 2 --gpu 0,0

Reply:
That will already work without the --gpu. @yhwen to confirm.

Reply:
I can check this later